src/vueWrapper.ts

```diff
@@ -42,6 +45,21 @@ export class VueWrapper<T extends ComponentPublicInstance> {
     return this.vm.$el.parentElement
   }
 
+  private attachNativeEventListener(): void {
+    const vm = this.vm
+    if (!vm) return
```

Comment:
Calling `this.element` breaks in some circumstances that I had trouble debugging, for some reason `findComponent` creates Wrapper objects without a vm? Anyway, if there's no vm, I guess there's nothing to bind event listeners to.

Reply:
I cannot imagine why this would happen 🤔  I will look at it

Reply:
Oh it makes sense, functional components can be found with `findComponent` but they do not have a `vm`

most likely someone is going to ask about capturing emitted events from functional components. let's deal with it if and when it comes up 🤷

Reply:
The test I added has a functional component for the grandparent, which works because it captures events that bubbled to the element of the out wrapper, I suppose. \it might be different for functional components without their own native DOM element but I suppose in that case DOM events wouldn't expect to be captured?